db/migrate/20160401043112_create_phones.rb

```diff
@@ -5,7 +5,7 @@ def change
 
       t.string :number, null: false
       t.string :extension
-      t.string :type, null: false
+      t.string :service_type, null: false
```

Comment:
I slightly prefer "type" to "service_type," mainly because it's not clear what "service" refers to. But I don't have that strong a preference. Rename is fine.